tests/osquery/test_api_views.py

```diff
@@ -575,6 +574,17 @@ def test_update_file_category_without_name(self):
             "name": ["This field may not be blank."]
         })
 
+    def test_update_file_category_slug_conflict(self):
```

Comment:
Nitpick. Try to order the tests the same way for the create and update operation.